model/SearchEngine/Driver/Elasticsearch/QueryBuilder.php

```diff
@@ -174,6 +175,27 @@ public function getSearchParams(
         return $params;
     }
 
+    public function getBoolQuery(string $field, array $values, string $index, ?int $size = self::DEFAULT_SIZE): array
```

Comment:
We do not need this, we can just create a query that returns all the related tests using the existing ElasticSearch implementation. Lets delete this, please